packages/admin-ui/ui/public/locales/th/translation.json

```diff
@@ -0,0 +1,2011 @@
+  "reservations-table-description": "คำอธิบาย",
+  "reservations-table-created": "สร้าง",
+  "reservations-table-quantity": "ปริมาณ",
+  "search-modal-start-typing-to-search": "เริ่มพิมพ์เพื่อค้นหา...",
+  "search-modal-clear-search": "ล้างการค้นหา",
+  "search-modal-or": "หรือ",
+  "search-modal-to-navigate": "เพื่อนำทาง",
+  "search-modal-to-select-and": "เพื่อเลือกและ",
+  "search-modal-to-search-anytime": "เพื่อค้นหาได้ตลอดเวลา",
+  "templates-settings": "การตั้งค่า",
+  "templates-manage-the-settings-for-your-medusa-store": "จัดการการตั้งค่าสำหรับร้าน Medusa ของคุณ",
+  "transfer-orders-modal-info": "ข้อมูล",
+  "transfer-orders-modal-customer-is-already-the-owner-of-the-order": "ลูกค้าเป็นเจ้าของคำสั่งซื้ออยู่แล้ว",
+  "transfer-orders-modal-success": "สำเร็จ",
+  "transfer-orders-modal-successfully-transferred-order-to-different-customer": "โอนคำสั่งซื้อไปยังลูกค้ารายอื่นเรียบร้อยแล้ว",
+  "transfer-orders-modal-error": "ข้อผิดพลาด",
+  "transfer-orders-modal-could-not-transfer-order-to-different-customer": "ไม่สามารถโอนคำสั่งซื้อไปยังลูกค้ารายอื่นได้",
+  "transfer-orders-modal-transfer-order": "คำสั่งโอน",
+  "transfer-orders-modal-order": "คำสั่งซื้อ",
+  "transfer-orders-modal-current-owner": "เจ้าของปัจจุบัน",
+  "transfer-orders-modal-the-customer-currently-related-to-this-order": "ลูกค้าที่เกี่ยวข้องกับคำสั่งซื้อนี้ในปัจจุบัน",
+  "transfer-orders-modal-new-owner": "เจ้าของใหม่",
+  "transfer-orders-modal-the-customer-to-transfer-this-order-to": "ลูกค้าที่จะโอนออเดอร์นี้ไปที่",
+  "transfer-orders-modal-cancel": "ยกเลิก",
+  "transfer-orders-modal-confirm": "ยืนยัน",
+  "templates-edit-user": "แก้ไขผู้ใช้",
+  "templates-remove-user": "ลบผู้ใช้",
+  "templates-resend-invitation": "ส่งคำเชิญอีกครั้ง",
+  "templates-success": "สำเร็จ",
+  "templates-invitiation-link-has-been-resent": "ลิงก์คำเชิญถูกส่งอีกครั้ง",
+  "templates-copy-invite-link": "คัดลอกลิงก์คำเชิญ",
+  "templates-invite-link-copied-to-clipboard": "คัดลอกลิงก์คำเชิญไปยังคลิปบอร์ดแล้ว",
+  "templates-remove-invitation": "ลบคำเชิญ",
+  "templates-expired": "หมดอายุ",
+  "templates-pending": "รอดำเนินการ",
+  "templates-all": "ทั้งหมด",
+  "templates-member": "สมาชิก",
+  "templates-admin": "ผู้ดูแลระบบ",
+  "templates-no-team-permissions": "ไม่ได้รับอนุญาตจากทีม",
+  "templates-status": "สถานะ",
+  "templates-active": "ใช้งาน",
+  "templates-name": "ชื่อ",
+  "templates-email": "อีเมล",
+  "templates-team-permissions": "การอนุญาตของทีม",
+  "templates-confirm-remove": "คุณแน่ใจหรือไม่ว่าต้องการลบผู้ใช้รายนี้",
+  "templates-remove-user-heading": "ลบผู้ใช้",
+  "templates-user-has-been-removed": "ผู้ใช้ถูกลบออกแล้ว",
+  "templates-confirm-remove-invite": "คุณแน่ใจหรือไม่ว่าต้องการลบคำเชิญนี้",
+  "templates-remove-invite": "ลบคำเชิญ",
+  "templates-invitiation-has-been-removed": "คำเชิญถูกลบออกแล้ว",
+  "multiselect-choose-categories": "เลือกหมวดหมู่",
+  "domain-categories-multiselect-selected-with-counts_one": "{{นับ}}",
```

Comment:
Search for `{{นับ}}` I think it may cause issue. The template variable shouldn't be translated.

Reply:
I appreciate your help in spotting the incorrect translation. 🙏🏼
For this and https://github.com/medusajs/medusa/pull/5654#discussion_r1403935150, I'll push the fix this evening.

Reply:
Fixed by https://github.com/medusajs/medusa/pull/5654/commits/89161bb7c15fd57c652b4030df95c7474ec74974